source/helpers/hotfix.ts

```diff
@@ -15,10 +17,16 @@ export const updateHotfixes = cache.function(async (): Promise<string[][]> => {
 		return [];
 	}
 
-	return atob(content)
-		.trim()
-		.split('\n')
-		.map(line => line.split(','));
+	const storage: HotfixStorage = [];
+	const lines = atob(content).trim().split('\n');
+	for (const line of lines) {
+		const [featureID, unaffectedVersion, relatedIssue] = line.split(',');
```

Comment:
Maybe the line itself should be trimmed too just in case?

Reply:
Maybe
```suggestion
		const [featureID, unaffectedVersion, relatedIssue] = line.split(/\s*,\s*/);
```